server/src/main/java/fr/uga/l3miage/tp2/exo3/models/Client.java

```diff
@@ -0,0 +1,19 @@
+package fr.uga.l3miage.tp2.exo3.models;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+public class Client {
+    @Id
+    private Long id;
+    private String nom;
+    private String email;
+
+    @OneToMany(mappedBy = "client")
+    private Set<Commande> commandes;
+
+    @OneToOne(mappedBy = "client")
+    private Adresse adresse;
```

Comment:
cette relation est unidirectionnel, et non bidi
```suggestion
    @OneToOne
    private Adresse adresse;
```